SECEdgar/base.py

```diff
@@ -80,7 +81,7 @@ def _handle_error(response):
             response(requests.response): Response object.
 
         Raises:
-            EDGARQueryError: If response throws error.
+            EDGARQueryError: If response returns error code.
```

Comment:
Python has changed a lot :( Initially, I thought it is not a valid Python:`returns error code` should be `returns error.code`

Reply:
But if it runs then it is fine.